src/Components/HomeCards/HomeCards.module.css

```diff
@@ -2,7 +2,8 @@
   display: grid;
   grid-template-columns: 50% 50%;
   grid-template-rows: 50% 50%;
-  width: 100%;
+  width: 100vw;
+  height: 100vw;
```

Comment:
Too many Volkswagens mate, maybe there is a similar unit that goes for height 🤔

Reply:
This was my way to try to keep the squares square, no matter the size of the screen. I have the same issue again with another component now. Tried like a million different solutions. Do you have an idea?